go.mod

```diff
@@ -14,13 +14,14 @@ require (
 	github.com/opencontainers/image-spec v1.1.0-rc6
 	github.com/pbnjay/memory v0.0.0-20210728143218-7b4eea64cf58
 	github.com/pterm/pterm v0.12.79
+	github.com/stretchr/testify v1.8.4
 	golang.org/x/mod v0.17.0
-	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.14.2
 	k8s.io/api v0.29.2
 	k8s.io/apimachinery v0.29.2
 	k8s.io/client-go v0.29.2
+	k8s.io/kubectl v0.29.0
```

Comment:
I take it the client-go is missing something?

Reply:
I'm not sure why this (and yaml) moved from indirect to direct depenencies. I just let `go mod tidy` do its thing